feature-utils/poly-look/src/css/index.js

```diff
@@ -7,6 +7,7 @@ import "./font.css";
 import "./layout.css";
 import "./design-system-tokens/radius.css";
 import "./design-system-tokens/spacing.css";
+import "./design-system-tokens/colors.css";
```

Comment:
IIRC one of the interesting things about tokens was the possibility of testing them independently. Are we going to set up something like that? Is it planned?

Reply:
You mean testing that the tokens under `design-system-tokens` are identical to those in Figma? I think we'll probably want to look into that as soon as we add some automation for getting the tokens - if we test that automation, we can be reasonably sure that the tokens are all present and have the desired values.

Reply:
Here is what we tested with @mteresa-jimenez at a very basic level
https://wiki.polypoly.eu/display/PN/Cross+guild+Design+System

[Figma tokens](https://www.figma.com/community/plugin/843461159747178978) is a plugin on Figma that gathers all tokens into a JSON (and can be synced on a github repository).
[Style dictionary](https://github.com/amzn/style-dictionary) is a tool that converts the json into variables of any language 

One thing that would be great is to have that JSON synced with confluence as well as Figma since that is where we are documenting polyLook. 
Happy to discuss this further